repo/packages/M/msoms/2/marathon.json.mustache

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "{{msoms.app_id}}",
+  "cpus": 1,
+  "mem": 256,
+  "disk": 0,
+  "instances": {{msoms.instances}},
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/var/lib/docker/containers",
+        "hostPath": "/var/lib/docker/containers",
+        "mode": "RW"
+      }
+    ],
+    "docker": {
+      "image": "{{resource.assets.container.docker.image}}",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 25225,
+          "hostPort": 25225,
+          "protocol": "tcp",
+          "labels": {}
+        }
+      ],
+      "privileged": true,
+      "parameters": [
+        {
+          "key": "restart",
+          "value": "always"
+        }
+      ],
+      "forcePullImage": true
+    }
+  },
+  "env": {
+    "WSID": "{{msoms.wsid}}",
+    "KEY": "{{msoms.wskey}}"
+  },
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "acceptedResourceRoles": [
+    "slave_public",
+    "*"
+  ],
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": {
+        "value": "status=$(ps -ef | grep -v grep | grep omsagent | wc -l); if ($status > 0); then exit 0; else exit 1; fi"
```

Comment:
this is a ridiculous health check... the presence of a process doesn't make it healthy

Reply:
"ridiculous" may be too strong a comment... but it doesn't make sense IMO.   Is there a better option?

Reply:
This is unrelated to this pull request. Right now we dont have other options to do the health check. We'll have to address it as a separate issue.